include/utils.hpp

```diff
@@ -358,5 +358,20 @@ bool executePreAction(const nlohmann::json& i_parsedConfigJson,
                       const std::string& i_vpdFilePath,
                       const std::string& i_flagToProcess);
 
+/**
+ * @brief Get EEPROM path from config JSON
+ *
+ * Given either D-bus inventory path/FRU EEPROM path,
+ * this API returns the primary EEPROM path fetched from JSON.
+ *
+ * @param[in] i_jsonObj - Parsed VPD config JSON object
+ * @param[in] i-path - Path in which VPD is present(D-bus path/FRU EEPROM path)
+ *
+ * @return On success returns valid primary EEPROM path, on failure returns
```

Comment:
nit: remove primary
nit FRU path

On return , what about exception , do you want to pass it back or handle it inside the function ?

Reply:
exceptions i want to pass back  to the caller.